Add Full Screen link to customizer's CSS editor.

jeherve opened the pull request:
I thought it would be nice to use @folletto's mockup and add a full-screen button to the corner of the new editor, linking to the Legacy one.

![screen shot 2016-12-16 at 13 10 12](https://cloud.githubusercontent.com/assets/426388/21262329/1147b334-c391-11e6-884a-d93badf3fd72.png)

Here is the end result I got to:

![screen shot 2016-12-16 at 14 48 17](https://cloud.githubusercontent.com/assets/426388/21264987/7a13a9e6-c39f-11e6-9dda-0f40e4b578a0.png)

### In this PR

- Enqueue a small JS and a small CSS file to add the button to the customizer.
- Update version numbers.
- Update readme.
- Create plugin version constant for easier versioning of enqueued assets.

### To be improved

I assume my .js file is just ugly. It works, but there is most likely a cleaner way to do things. I should also probably add that CSS inline on the page instead of enqueuing a new file.

jeherve committed:
Add Full Screen link to customizer's CSS editor.

- Enqueue a small JS and a small CSS file to add the button to the customizer.
- Update version numbers.
- Update readme.
- Create plugin version constant for easier versioning of enqueued assets.

folletto commented:
I didn't think of that, it's a clever idea to anticipate the feature! :D 

Any chance we can style it on gray background as the design mock?

jeherve committed:
Change color of the customizer link bar

@see https://github.com/georgestephanis/legacy-jetpack-custom-css-editor/pull/3#issuecomment-267601057

jeherve commented:
Anything is possible :)

![screen shot 2016-12-16 at 15 21 41](https://cloud.githubusercontent.com/assets/426388/21265822/77f66730-c3a3-11e6-8cf9-528d404b7e4a.png)

You'll notice that the spacing is different, though. That's because we now have notices that can be displayed below the title, and before the editor.

folletto commented:
Cool! :D 
Center it vertically too, and design wise it's 👍

jeherve commented:
That's a bit trickier as there can be notices above the button:

![screen shot 2016-12-16 at 15 45 22](https://cloud.githubusercontent.com/assets/426388/21266485/b705be96-c3a6-11e6-85fc-c3b18c6d63e8.png)

If I vertical align now, it won't be when the notices are displayed.

folletto commented:
In theory... (theory)... it should be vertically aligned without notice in that space, and also get in the same vertical alignment when that notification is in (in that same space)... but I guess I should check the code :P
